1208-projects/my_projects.feature

```diff
@@ -1,74 +1,78 @@
-    I want to be able to search in My Projects
+    Background: 
+        Given A User is logged in
 
     @2874a
+    @updated
     Scenario: User opens My Projects
-        When they navigate to the My Projects Page
-        Then they see a Create New Project Button
-        And they see a search field
-        And they see a list of Active Projects
-        And they see a collapsed list of Concluded Projects
-        And the lists contains Projects where the User has the role as:
+        When a User navigate to My Projects Page
+        Then the User see a option to create a new project
+        And the User see a search field
+        And the User see a list of Active Projects
+        And the User see a collapsed list of Concluded Projects
+        And the lists contains Projects where the User has one of the following roles:
             | Project Owner         |
             | Project Manager       |
             | Local Project Manager |
             | Participants          |
-        And each Project has:
-            | Title           |
-            | Institution     |
-            | Project Manager |
-        And each Project has an Edit button if the User has role: 
+        And each Project views it's:
+            | Title                    |
+            | Coordinating Institution |
+            | Project Manager          |
+        And each Project can be edited if the User has one of the following roles: 
             | Project Owner         |
             | Project Manager       |
             | Local Project Manager |
         And the list can be sorted by:
-            | Title           |
-            | Institution     |
-            | Project Manager |
-        And they see pagination buttons for the Concluded Projects list
+            | Title                    |
+            | Coordinating Institution |
+            | Project Manager          |
+        And the User can navigate through the possible long list of Concluded Projects
     
     @2874b
     Scenario: User opens a Project's Landing Page
         Given User opens My Projects
-        When they click a Project's Title 
-        Then they see the Landing Page for the Project
+        When the User click a Project's Title 
+        Then the User see the Landing Page for the Project
 
     @2874c
-    Scenario Outline: User Edits a Project in the Project Wizard
-        # See also Curator opens a Project in the Project Wizard
-        Given User opens My Projects
-        And User has "<Role>" on the Project
-        And the project lacks an Approval of type "REK"
-        # A "REK" Approved project is a Health Project. 
-        # See health_related_projects.feature for details.
-        When they click the Edit Button 
-        Then they see the Project in the Project Wizard
-        Examples:
-            | Role                  |
+    Scenario: User Edits a Project in the Project Wizard
+        When a User edits a project from My Projects
```

Comment:
Ser ut som det er litt feil rekkefølge på ting her
```suggestion
        When a User clicks the Edit Button for a project from My Projects
```

Reply:
I have rearranged the roles into a Given. Not included "the Edit Button" to allow designer to decide how the user chooses to edit a project